code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -32,7 +32,7 @@ BONUS
 			if(1, 2, 3)
 				to_chat(M, "<span notice='notice'>[pick("You swallow excess mucus.", "You lightly cough.")]</span>")
 			else
-				M.emote("cough")
+				M.emote("cough", , , 1)
```

Comment:
Oh right, I thought you meant within the same type.
Yeah, those overrides should be preserving arguments, anything else is bad standards.
New() being the only exception.